ratik/src/main/java/org/opencadc/ratik/Main.java

```diff
@@ -169,17 +170,18 @@ public static void main(final String[] args) {
             final String configuredQueryService = props.getFirstPropertyValue(QUERY_SERVICE_CONFIG_KEY);
             final URI resourceID = URI.create(configuredQueryService);
 
+            final String configuredFileService = props.getFirstPropertyValue(FILE_SERVICE_CONFIG_KEY);
+            final StorageSite remoteSite = new StorageSite(URI.create(configuredFileService), configuredFileService,
```

Comment:
The change in InventoryValidator to have a ref to the remote StorageSite (instead of a boolean) did not mean that this should propagate all the way to main and config file content. In fact, the config should be as it was (and like fenwick) because making the remote files service configurable just increases the chance of misconfiguring ratik. Plus, how would one configure for running ratik in a storage site and validating against a remote global?

The config structure was correct already. Like fenwick, ratik has to query for and assign the StorageSite **if trackSiteLocations  is true** and leave it null if false.

Reply:
Removed the files resourceID from the config file, and query the remote site for the StorageSite if running on global.